src/main/java/com/xiaomi/infra/pegasus/security/SaslWrapper.java

```diff
@@ -51,4 +52,17 @@
               return saslClient.getMechanismName().getBytes(Charset.defaultCharset());
             });
   }
+
+  public blob getInitialResponse() throws PrivilegedActionException {
```

Comment:
This is function name is not clear enough. What does that mean?

Reply:
I added a comment in front of func getInitialResponse. pls have a look